bonsai/ethereum/contracts/callback_dummy.sol

```diff
@@ -15,29 +15,29 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.17;
 
-import "./proxy.sol";
+import "./BonsaiTestRelay.sol";
```

Comment:
I recommend importing and using the interface instead of `BonsaiTestRelay`.
```suggestion
import {IBonsaiRelay} from "./IBonsaiRelay.sol";
```

Reply:
Resolved in https://github.com/risc0/risc0/pull/738/commits/c354c6ffd4c25411860b768de83dc8266ea89da9